src/components/interfaces/HMI_API.xml

```diff
@@ -2356,8 +2356,8 @@
       <param name="fileName" type="String" maxlength="255" minlength="1" mandatory="true">
           <description>The path to file.</description>
       </param>
-      <param name="appID" type="Integer" mandatory="false">
-        <description>ID of application that requested this RPC.</description>
+      <param name="appID" type="Integer" mandatory="true">
```

Comment:
@dtrunov fixed in [443f98e](https://github.com/smartdevicelink/sdl_core/pull/1082/commits/443f98ee2cbd27d3a33bb6d97907dd8243898f52)